src/DashboardService.Models.Dto/Requests/Task/PatchTaskRequest.cs

```diff
@@ -0,0 +1,12 @@
+using System;
+
+namespace LT.DigitalOffice.DashboardService.Models.Dto.Requests.Task;
+
+public record PatchTaskRequest
+{
+  public Guid? TaskTypeId { get; set; }
+  public Guid? PriorityId { get; set; }
+  public string Name { get; set; }
+  public string Content { get; set; }
+  public DateTime DeadlineAtUtc { get; set; }
```

Comment:
You can create Task with null deadline(CreateTaskRequest). But can not change it to null?

Reply:
++